setup.py

```diff
@@ -51,7 +51,6 @@
     "msrest>=0.6.21",
     "msrestazure>=0.6.3,<2.0.0",
     "jsonschema~=3.2.0",
-    "importlib_metadata;python_version<'3.8'",
```

Comment:
this is corresponding usage of the backcompat importlib_metadata https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/common/utility.py#L372

Reply:
alright will remove